libvast/src/system/spawn_source.cpp

```diff
@@ -30,6 +31,10 @@ spawn_source(node_actor::stateful_pointer<node_state> self,
                            "unable to spawn a remote source when spawning a "
                            "node locally instead of connecting to one; please "
                            "unset the option vast.node");
+  auto transforms
+    = make_transforms(transforms_location::client_source, args.inv.options);
```

Comment:
This might need to be `transforms_location::server_import`, needs follow-up discussion.